### PR TITLE
feat: report-problem base

### PR DESCRIPTION
**Pull request by NightmareFuryRemi**

### CODE REVIEW

**Review by EvilProfesseur — CHANGES_REQUESTED** [review]

**Review by krzysztofkleba — COMMENTED** [review]
just to be marked as reviewer

File: frontend/src/shared/components/report-problem/reportProblem.css
Comment (krzysztofkleba) [inline frontend/src/shared/components/report-problem/reportProblem.css:1]: try to remove  `frontend/src/shared/components/report-problem/reportProblem.css` and `frontend/src/shared/components/report-problem/reportProblem.min.css` -> then see if your app still working, if yes we dont need to push them into repository and simply this files are redundant.

File: frontend/src/shared/services/reportProblem.service.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+export function userReportProblem() {
```
Comment (krzysztofkleba) [inline frontend/src/shared/services/reportProblem.service.ts:4]: rename it to `useReportProblem`

**Review by krzysztofkleba — CHANGES_REQUESTED** [review]

**Comment by EvilProfesseur** [comment]
Controls should be in line with the body of the component. 
image

**Comment by EvilProfesseur** [comment]
"Describe the problem" text is displayed in a wrong font

**Review by EvilProfesseur — APPROVED** [review]

**Pull request merged**